pkgdb/src/registry/floxpkgs/flake.nix.in

```diff
@@ -0,0 +1,270 @@
+# -*- mode: nix; -*-
+# ============================================================================ #
+#
+# Wraps `nixpkgs' to patch the package set for traversal by `pkgdb scrape'.
+#
+# !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+#
+# Packages should NEVER be consumed/built from this flake!
+# It exists to produce databases for `pkgdb'!
```

Comment:
question: is this comment still accurate? How do we build unfree packages then?

Reply:
It's not accurate, I never updated the comments from Alex's branch.